src/main/java/se/skvf/kaninregister/api/BunnyRegistryApiImpl.java

```diff
@@ -392,14 +392,33 @@ private BunnyList toBunnyList(Collection<Bunny> bunnies) {
 		list.setBunnies(bunnies.stream().map(BunnyRegistryApiImpl::toListDTO).collect(Collectors.toList()));
 		return list;
 	}
-	
+
+	@Override
+	public OwnerDTO session() {
+		return process(() -> {
+			String session = getSession();
+			if (session == null) {
+				return null;
+			}
+
+			String ownerId = sessions.getOwnerIdForSession(session);
+			if (isBlank(ownerId)) {
+				return null;
+			}
+
+			return toDTO(validateOwner(ownerId, false));
+		});
+	};
+
 	@Override
 	public OwnerDTO login(LoginDTO loginDTO) {
 		return process(() -> {
 
-			if (getSession() != null) {

```

Comment:
Om servern startar om och klienten har en kaka kvar så kunde man inte logga in

Reply:
Denna kanske inte borde ha ett returvärde som någon då något annat i browsern kan plocka upp genom att göra anropet. Se min chat om att upptäcka HttpOnly-kakor i JS (för innehållet är ju ändå inte intressant).

Reply:
/sessions-anropet tog ju ganska lång tid att göra. Kanske kan prestandaoptimera lite genom att först kolla om cookie finns och det är först när det finns en cookie som /session anropas så att den ev pågående sessionen kan återstartas vid refresh/F5 av browsern. 
Det kan jag kolla lite på i kväll. Men hur ska vi hantera de som har en pågående session (och cookie i browsern) när servern uppdateras/startas om?

Reply:
Jag committade precis en uppdatering och pushade in till denna PR. Om det inte finns en cookie kollas med knepet som du tipsade om Jonas. Om det finns en cookie försöker jag hämta upp innehåll i sessionen om det finns en session.